packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
do we need version bump to the interface package? @stuartmorgan

Reply:
This doesn't match pubspec.yaml, and is failing CI accordingly.